### PR TITLE
Adds short day of the week to the date, above my-units calendar

### PR DESCRIPTION
**Pull request by vincit-matu**
## 🛠️ Changelog
- Adds the short day name next to the date, above the calendar in my-units section (admin side)
- Also attempts to add the same favicon which is in use on the ui/client-side, in exactly the same way - however it doesn't seem to have intended consequences (..or any at all 😮)

## 🧪 Test plan
- Check out the calendar in a reservation unit page, under my units.
- See if anything sticks out to you regarding why the favicon isn't showing as on the UI-side

## 🎫 Tickets

- TILA-3010

### CODE REVIEW

**Review by joonatank — CHANGES_REQUESTED** [review]
Doesn't work, like we talked.  Just marking so it doesn't get merged accidentially.

**Comment by vincit-matu** [comment]
> Doesn't work, like we talked. Just marking so it doesn't get merged accidentially.

Yup, the date input is broken all-in-all. Moved the weekday outside the `input`-element, but it would need design decisions and then styling according to those before being ready.

File: apps/ui/components/reservation-unit/Head.tsx
Comment (joonatank) [inline apps/ui/components/reservation-unit/Head.tsx:1]: Got an extra commit here from the other PR :)

Comment (vincit-matu) [reply apps/ui/components/reservation-unit/Head.tsx:1]: Hehe, fixed! 😊

**Pull request merged**